R/batchAnnotation.R

```diff
@@ -109,7 +117,7 @@ addFileLink <- function(batchCodeList, recordedBy, experiment, lsTransaction,
   }
   locationState <- existingStates[[1]]
   
-  if (racas::applicationSettings$server.service.external.file.type == "blueimp") {
+  if (racas::applicationSettings$server.service.external.file.type %in% c("blueimp", "gcs")) {
```

Comment:
This applies to both our known file storage types

Reply:
Not to "custom" though?